.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: python
+python:
+  - "2.7"
+  - "3.4"
+  - "3.5"
+os:
+  - linux
+# ** patchelf isn't supported here, _yet_ **
+# addons:
+#   apt:
+#     sources:
+#       - ppa:utlemming/rax-playground
+#     packages:
+#       - build-essential
+#       - ninja-build
+#       - patchelf
+#       - libboost-dev
+#       - libboost-program-options-dev
+#       - libogg-dev
+before_install:
+    - sudo add-apt-repository ppa:utlemming/rax-playground -y
+    - sudo apt-get update -qq
+    - sudo apt-get install build-essential ninja-build libboost-dev libboost-program-options-dev libogg-dev patchelf
+install: pip install flake8 .
```

Comment:
I think you'll need to add `python setup.py install` here. The integration tests assume that you've at least called `develop` right now, since they run the `bfg9000` script directly. Removing that requirement would be hard, since the "compiled" build scripts need to refer to the installed copies of `bfg9000` and related scripts (e.g. `bfg9000-depfixer`).